.github/workflows/build.yml

```
@@ -28,6 +28,10 @@ jobs:
         with:
           node-version: 18
           cache: 'npm'
+      - name: Get Commit Hash
+        run: echo "COMMIT_HASH=$(git rev-parse --short HEAD)" >> $GITHUB_ENV
+      - name: Generate Version File
+        run: echo "export const COMMIT_HASH = '${{ env.COMMIT_HASH }}';" > src/version.js
```

Won't it be easier to do something like this?
```suggestion
        run: echo "$(git rev-parse --short HEAD)" >> public/hash.txt
```